js/src/carousel.js

```diff
@@ -223,6 +225,22 @@ const Carousel = (($) => {
       return config
     }
 
+    _getTransitionDuration() {
+      // Get transition-duration of first element in the carousel
+      let transitionDuration = $(this._element).find(Selector.ITEM).css('transition-duration')
+
+      // Return 0 carousel item is not found
+      if (!transitionDuration) {
+        return 0
+      }
+
+      // If multiple durations are defined, take the first
+      transitionDuration = transitionDuration.split(',')[0]
+
+      // Multiply by 1000 if transition-duration is defined in seconds
+      return transitionDuration.indexOf('ms') > -1 ? parseFloat(transitionDuration) : parseFloat(transitionDuration) * MILLISECONDS_MULTIPLIER
```

Comment:
Can you break this line?

Reply:
I had that before, but this caused a linting error:
https://travis-ci.org/twbs/bootstrap/jobs/328121951

Maybe store `parseFloat(transitionDuration)` in a variable to save some space?